TAO/orbsvcs/tests/CosEvent/Basic/Random.cpp

```diff
@@ -344,6 +344,12 @@ RND_Consumer::push (const CORBA::Any &event)
 }
 
 void
+RND_Timer::push (const CORBA::Any &event)
```

Comment:
tried a test compile, but it seems like I've spoiled my ACE build tree with a "wrong-env" make. might take a bit longer than expected, but once the compile passes the stage where Random.o gets linked I'll prepare the commit & PR.